sdk/kubeflow/trainer/types/types.py

```diff
@@ -45,10 +45,46 @@ class CustomTrainer:
     resources_per_node: Optional[Dict] = None
 
 
+# Configuration for the TorchTune LLM Trainer.
+@dataclass
+class TorchTuneConfig:
+    """TorchTune LLM Trainer configuration. Configure the parameters in
+        the TorchTune LLM Trainer that already includes the fine-tuning logic.
+
+    Args:
+        dtype (`Optional[str]`):
+            The underlying data type used to represent the model and optimizer parameters.
+            Currently, we only support `bf16` and `fp32`.
+        batch_size (`Optional[int]`):
+            The number of samples processed before updating model weights.
+        epochs (`Optional[int]`):
+            The number of samples processed before updating model weights.
+        loss (`Optional[str]`): The loss algorithm we use to fine-tune the LLM,
+            e.g. `torchtune.modules.loss.CEWithChunkedOutputLoss`.
+        num_nodes (`Optional[int]`): The number of nodes to use for training.
+        resources_per_node (`Optional[Dict]`): The computing resources to allocate per node.
+    """
+
+    dtype: Optional[str] = None
+    batch_size: Optional[int] = None
+    epochs: Optional[int] = None
+    loss: Optional[str] = None
```

Comment:
I think, we should make loss enums as well, right ?
https://github.com/pytorch/torchtune/blob/main/recipes/configs/llama3_3/70B_lora.yaml#L58-L59